python/isort_select/files/base

```diff
@@ -0,0 +1 @@
+bin/isort
```

Comment:
I think we should add here the other console_script as well: `bin/isort-identify-imports` (you can see what gets installed with `port contents pyXY-isort`.